spec/swagger.yaml

```diff
@@ -408,7 +410,7 @@ definitions:
         description: The secondary key value of the record
       result:
         type: string
-        example: 204
+        example: "204"
```

Comment:
Swagger UI will show this as an int instead of a String if it's not explicitly quoted in the example.